src/Views/Spinner.js

```diff
@@ -1,6 +1,7 @@
+import { settings} from '../js/component/Variable';
```

Comment:
'component/Variable' better to place it as a separate file not in scope of components. File with constants should be named as constants.js 
All files which are related to components/classes should be named with the first letter in upper case, all other files in lover case:
examples - App, Spinner, NewsChannel but helpers, constants, selectors....